sway-lib-std/src/u256.sw

```diff
@@ -52,6 +52,21 @@ impl From<(u64, u64, u64, u64)> for U256 {
     }
 }
 
+impl From<u64> for U256 {
+    fn from(num: u64) -> U256 {
+        U256 {
+            a: 0,
+            b: 0,
+            c: 0,
+            d: num,
+        }
+    }
+
+    fn into(self) -> u64 {
+        self.as_u64().unwrap()
+    }
```

Comment:
Here, `into` may fail, but the Rust semantics says that the `From`/`Into` trait methods must not fail (see https://doc.rust-lang.org/std/convert/trait.From.html).